README.md

```diff
@@ -223,7 +223,9 @@ Properly counting such votes is a missing feature.
 The bot may be started like any node.js script. For example:
 
 ```
-node Main.js /etc/anubis/config.json
+git clone git@github.com:measurement-factory/anubis.git
+cd ./anubis
```

Comment:
I do not think this recipe works well because it skips installation of dependencies (e.g., `npm update`). It is rather unlikely that somebody will have _all_ the right dependencies installed globally. However, I doubt we should tell people how to install Javascript-based software -- there are just too many environment-specific ways to do that.

I suggest fixing the invocation line (like you did) but not giving any other details like `git clone`. The invocation line itself _is_ useful IMO because it is not obvious which `.js` file the user should run or how to specify the configuration file.